.github/workflows/wan-perf.yml

```diff
@@ -73,6 +73,7 @@ jobs:
         rtt: [5, 50, 200, 500]
         queueRatio: [0.2, 1, 5]
         loss: [0,1000,10000]
+        congestionControl: ["cubic", "bbr"]
```

Comment:
GH Actions doesn't like this, it produces too many unique jobs. Please put this in the `env` instead.

Reply:
moved to env node